App.tsx

```diff
@@ -1,19 +1,51 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+// https://github.com/software-mansion/react-native-gesture-handler/issues/320#issuecomment-443815828
+import 'react-native-gesture-handler';
+
+import { createStackNavigator } from '@react-navigation/stack';
+import * as React from 'react';
+import { StyleSheet, YellowBox } from 'react-native';
+import { AppearanceProvider } from 'react-native-appearance';
+import { enableScreens } from 'react-native-screens';
+import { SafeAreaProvider } from 'react-native-safe-area-context';
+import { Provider, connect } from 'react-redux';
+// @ts-ignore
+import { PersistGate } from 'redux-persist/integration/react';
+// import QuickActions from 'react-native-quick-actions';
```

Comment:
Do you want to keep this comment or can we drop it?

Reply:
I want to keep it. We will use it later ✌️